src/components/Icon/Icons.jsx

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
where did you grab these paths from? will it be easy to find and generate them later?

Reply:
I generated them out of sketch via copy svg. Not sure how to get it back into sketch, but to render out an svg, you could paste it into an svg path tag.